src/remote/client/interface.cpp

```diff
@@ -9313,6 +9313,16 @@ static void receive_packet_noqueue(rem_port* port, PACKET* packet)
 				stmt_id = p->packet.p_sqlfree.p_sqlfree_statement;
 				bFreeStmt = (p->packet.p_sqlfree.p_sqlfree_option == DSQL_drop);
 				break;
+
+			case op_batch_blob_stream:
+				stmt_id = p->packet.p_batch_blob.p_batch_statement;
+				bCheckResponse = true;
+				break;
+
+			case op_batch_regblob:
+				stmt_id = p->packet.p_batch_regblob.p_batch_statement;
+				bCheckResponse = true;
+				break;
```

Comment:
Looks like all `op_batch_XXX` labels have the same handler, why not unite them ?

Reply:
Now it is clear, thanks